### PR TITLE
For #8700 - Do not tint TrackingProtectionIconView when animatable

### PR DESCRIPTION
**Pull request by ekager**
### Pull Request checklist

- [ ] **Quality**: This PR builds and passes detekt/ktlint checks (A pre-push hook is recommended)
- [ ] **Tests**: This PR includes thorough tests or an explanation of why it does not
- [ ] **Changelog**: This PR includes [a changelog entry](https://github.com/mozilla-mobile/android-components/blob/master/docs/changelog.md) or does not need one
- [ ] **Accessibility**: The code in this PR follows [accessibility best practices](https://github.com/mozilla-mobile/shared-docs/blob/master/android/accessibility_guide.md) or does not include any user facing features

### After merge
- **Milestone**: Make sure issues closed by this pull request are added to the [milestone](https://github.com/mozilla-mobile/android-components/milestones) of the version currently in development.
- **Breaking Changes**: If this is a breaking change, please push a draft PR on [Reference Browser](https://github.com/mozilla-mobile/reference-browser) to address the breaking issues.

### CODE REVIEW

File: components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/TrackingProtectionIconView.kt [hunk 1]
```diff
@@ -68,6 +76,10 @@ internal class TrackingProtectionIconView @JvmOverloads constructor(
             null
         }
 
+        if (update.drawable is Animatable) {
+            clearColorFilter()
+        } else trackingProtectionTint?.let { setColorFilter(it) }
+
```
Comment (Amejia481) [inline components/browser/toolbar/src/main/java/mozilla/components/browser/toolbar/display/TrackingProtectionIconView.kt:82]: Could we also add a test to cover this section 🤓 ?

**Review by Amejia481 — COMMENTED** [review]
Looks great, lets just add an additional test and an entry to the changelog :)

**Review by Amejia481 — APPROVED** [review]
👍🏽

**Comment by ekager** [comment]
bors r=Amejia481

**Pull request merged**